R/screenshot.R

```diff
@@ -0,0 +1,60 @@
+#' A function to make screenshots from an OTTR bookdown website
+#' @description This function creates screenshots of course chapters that are stored in a created output directory 
+#' @param git_pat default is NULL; required argument; a Git secret
+#' @param repo default is NULL; required argument; GitHub repository name, e.g., jhudsl/OTTR_Template
+#' @param output_dir default is "resources/chapt_screen_images"; Output directory where the chapter's screen images should be stored
+#' @param base_url default is NULL; rendered bookdown URL where screenshots are taken from, if NULL, the function will use the repo_name and and git_pat to find the base_url
```

Comment:
Had to edit this description to change it from the original in the script. How is it?

Reply:
Looks like all the essentials are here! Only thing you could add if you wanted to be extra nice is a link to the page in GitHub about PATs: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens